### ISSUE TITLE
does not run with node 12.18.2, runs with node 15.11.0 while package.json says >8.9.1

### ISSUE THREAD

**Issue by gruzilla** [colyseus/schema#105]
on node v12.18.2 and node v8.9.4 when running

```
npm init colyseus-app ./my-colyseus-app
cd my-colyseus-app
npm start
```

i get the following error message:
```

/node_modules/@colyseus/schema/build/cjs/index.js:428
        const operation = this.$changes.indexes[index]?.op ?? exports.OPERATION.ADD;
                                                       ^
```

i tracked the change to this commit:
https://github.com/colyseus/schema/commit/7e01256374950c853275f86cd41842928b01c58f#diff-ce63faa84c2addc214b67b926cc8bf957fa9593c9eb3bca94a55652dd30f3e9bR162

when i switch to node 15.11.0 it works.

please highlight somewhere, that this node version is required as it was not obvious to me and cost me some hours.

**Comment by endel** [comment]
Thank you for reporting @gruzilla! 

This problem has been introduced a few days ago with the addition of ESM support (https://github.com/colyseus/schema/pull/104)

The problem is that the CJS version of `@colyseus/schema` had JavaScript code not compatible with old Node.js versions - which has been fixed now on version `1.0.16` (https://github.com/colyseus/schema/commit/dd90e254855af7f0c1f20715c2ce8db58cec0c91)

Cheers, please let me know if you run into any other issues!

**Comment by gruzilla** [comment]
thank you for your very quick reply and also for explaining!
